sdk/src/client/api/high_level.rs

```diff
@@ -30,9 +29,10 @@ impl Client {
     pub async fn inputs_from_transaction_id(&self, transaction_id: &TransactionId) -> Result<Vec<OutputWithMetadata>> {
         let block = self.get_included_block(transaction_id).await?;
 
-        let inputs = match block.payload() {
+        let inputs = match block.block().payload() {
             Some(Payload::Transaction(t)) => t.essence().inputs(),
             _ => {
+                // TODO ???
```

Comment:
Well since this is by transaction ID it really should be unreachable

Reply:
Fair enough

Reply:
Done